### PR TITLE
Small data preparation changes (ensure sufficient active molecules in random test/val splits; do not resplit if data already loaded) and set reasonable default hyperparameters

### PR DESCRIPTION
**Pull request by willy-b**
- Ensure we do not randomly get too few active molecules in the test and validation splits 
  (we have very few active molecules, it is easy to randomly miss including them in a random test split)

- Reuse existing data if already loaded in OGB compatible form from raw PCBA CSV
  this allows, doing multiple training runs with randomized weights on the same data split to get performance statistics for hyperparameter values on consistent data (now the default behavior after this change)

- Note that I will almost certainly need to get additional data beyond PCBA AID 577 as there are just not enough active molecules to train, validate, and test on to learn the task, I believe.  This is just a mitigation to ensure a bare minimum while I scout out new datasets, like e.g. maybe https://pubchem.ncbi.nlm.nih.gov/bioassay/1621 which also has too few actives to use alone, so I may combine multiple such datasets. Combining multiple such dataset may make the task harder to learn though as they are measuring different things (antiviral activity through unknown mechanism vs specifically NS2bNS3 proteinase inhibition).

- Adjust hyperparameters for extremely rare positive class data regime (<0.2% active molecules, < 100 active molecules in train split to learn from)  
    - increase default batch size to 1024 as needs larger batches since < 0.2% of the data is active molecules otherwise almost all batches will be entirely inactive examples for that gradient descent step
    
    - increase default weight decay to 5e-4 as we are learning from very few examples (<100 active examples total in the training data), so we increase regularization to avoid overfitting

### CODE REVIEW

**Comment by willy-b** [comment]
(Force pushed this off-main unmerged branch to improve the commit message on the 2nd commit in this PR, no code change.)
